openverse_catalog/dags/providers/provider_api_scripts/phylopic.py

```diff
@@ -160,7 +160,9 @@ def _get_creator_details(result: dict) -> tuple[str | None, str | None, str | No
         submitter = result.get("submitter", {})
         first_name = submitter.get("firstName")
         last_name = submitter.get("lastName")
-        if first_name and last_name:
+        if first_name or last_name:
+            f"{first_name} {last_name}".strip()
```

Comment:
This is not actually being assigned to `creator` here. The conditional is also too broad, since this captures the case where `first_name`, `last_name`, or both names are defined.

I think what we're looking for here is to keep the original logic, which sets `creator` to `f"{first_name} {last_name}"` when both variables are defined, but add additional logic to set `creator` to `first_name` when _only_ the first name is defined, and vice versa for `last_name`.